css/index.css

```diff
@@ -3,6 +3,9 @@ body
 {
     background-color: lightblue; 
     color: red;
+}
+header{
+    position:sticky;
```

Comment:
The header isn't sticky in my browser. On the MDN page for [CSS positioning](https://developer.mozilla.org/en-US/docs/Web/CSS/position?v=example), it mentions:

> At least one [inset](https://developer.mozilla.org/en-US/docs/Web/CSS/inset) property ([top](https://developer.mozilla.org/en-US/docs/Web/CSS/top), [inset-block-start](https://developer.mozilla.org/en-US/docs/Web/CSS/inset-block-start), [right](https://developer.mozilla.org/en-US/docs/Web/CSS/right), [inset-inline-end](https://developer.mozilla.org/en-US/docs/Web/CSS/inset-inline-end), etc.) needs to be set to a non-auto value for the axis on which the element needs to be made sticky. If both inset properties for an axis are set to auto, on that axis the sticky value will behave as relative.

If we add a `inset: 0px` to the header's CSS, then the bar becomes sticky. This was a good attempt though. Take a look at the pages above and please let me know if you have more questions.